Safety/out-of-hours-support.rst

```diff
@@ -0,0 +1,88 @@
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Safety-out-of-hours-support:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+.. _`Emergency Call List`: https://start.1password.com/open/i?a=EJGC5QLBFNAJ3ODCNVC2I3HIPM&v=mqlr5qdkpipoyamzhty5kg365y&i=tu4v5aa4tvhbm4e3rc5siiwada&h=lsstit.1password.com
+.. _`download`: https://1password.com/downloads
+.. _`IT support`: https://jira.lsstcorp.org/projects/IHS/issues/IHS-6753?filter=allopenissues
+.. _`#auxtel-operations`: https://lsstc.slack.com/archives/CLQ63T89L
+.. _`#auxtel-virtual-control-room`: https://lsstc.slack.com/archives/C01K4M6R4AH
+.. _`#rubin-observing`: https://lsstc.slack.com/archives/C04HULH5HHD
+
+####################
+Out of hours support
+####################
+
+The Out of Hours support call list is managed in a 1Password vault named "Emergency Call List".
+All observing specialists are automatically enrolled in the vault once they have a 1Password account.
+You must submit a JIRA ticket to `IT support`_ to be enrolled in the Rubin 1Password system. 
+Once enrolled, you can `download`_ a desktop application or browser extension to use 1Password and access the information the `Emergency Call List`_ vault. 
+
+.. note::
+    If you volunteer to be on the Emergency Call List, you will see the phone numbers provided by every other volunteer.
+    Your number will also be accessible to everyone with permissions to the vault.
+    If you volunteer, please provide your availability to the best of your knowledge, including holdiays and weekends.
+
+.. _out-of-hours-support-procedure:
+
+Procedure
+^^^^^^^^^
+
+- If a fault happens during the night, or some new behavior appears in the system, observers and summit staff should attempt to troubleshoot to the extent of their knowledge.
+
+- Ask for help on Slack:
+    - `#auxtel-operations`_
+    - `#auxtel-virtual-control-room`_
+    - `#rubin-observing`_
+
+- If no response is received from Slack, and summit staff have run out of ideas or need assistance, activate the Emergency Call List.
```

Comment:
activate  == consult ??

I think you should remind people to first consult the confluence page about people's availability before calling. The link to the confluence page is also found inside the vault.

Reply:
That's correct... but you can say that there is a confluence link in the 1password vault.
The confluence page will be updated more than what is inside the vault (sadly).

Reply:
👍